ckan/config/config_declaration.yaml

```diff
@@ -461,6 +461,26 @@ groups:
         type: int
         default: 2
 
+  - annotation: Uploader Settings
+    options:
+      - key: ckan.upload.user.types
```

Comment:
Can you document these in `doc/configuration.rst`? IIRC this is still not automatically generated right?

Reply:
Yes. We can use tokens that will be replaced declared defaults, but it must be done manually